src/libraries/System.Formats.Tar/src/System/Formats/Tar/GnuTarEntry.cs

```diff
@@ -112,7 +111,6 @@ public DateTimeOffset ChangeTime
             get => _header._cTime;
             set
             {
-                ArgumentOutOfRangeException.ThrowIfLessThan(value, DateTimeOffset.UnixEpoch);
```

Comment:
Should we have a different lower limit here?

Reply:
I'll update this per https://github.com/dotnet/runtime/pull/101172#discussion_r1609681536.

With the GNU numeric format there will be no lower range. With the other formats it will throw for less than `UnixEpoch` as before.

Reply:
Ok I see what you did here. Since aTime and cTime properties are only available in the GNU format, we don't need to throw for this specific UnixEpoch case, as we should be able to set these fields to MinValue.

And for PAX, those two fields get added directly to the extended attributes, there are no properties for them.